configure-system-libraries.sh

```diff
@@ -56,11 +56,19 @@ patch_config()
 }
 
 if [ "$(uname -s)" = "Darwin" ]; then
-	SEARCHDIRS="/usr/local/lib /usr/local/opt/openal-soft/lib"
-	patch_config "Lua 5.1" "${SEARCHDIRS}" bin/Eluant.dll.config lua51.dylib liblua5.1.dylib
-	patch_config SDL2 "${SEARCHDIRS}" bin/SDL2-CS.dll.config SDL2.dylib libSDL2-2.0.0.dylib
-	patch_config OpenAL "${SEARCHDIRS}" bin/OpenAL-CS.Core.dll.config soft_oal.dylib libopenal.1.dylib
-	patch_config FreeType "${SEARCHDIRS}" bin/OpenRA.Platforms.Default.dll.config freetype6.dylib libfreetype.6.dylib
+	if [ "$(arch)" = "arm64" ]; then
+		SEARCHDIRS="/opt/homebrew/lib /opt/homebrew/opt/openal-soft/lib"
+		patch_config "Lua 5.1" "${SEARCHDIRS}" bin/Eluant.dll.config lua51.dylib liblua5.1.dylib
```

Comment:
These patch_config lines look identical in the two branches, so we can avoid duplication by having just the SEARCHDIRS here.

Reply:
That is true. I've amended the commit.